cosmic-panel-config/Cargo.toml

```diff
@@ -12,8 +12,8 @@ anyhow = "1.0.68"
 ron = "0.8.0"
 serde = { version = "1.0.152", features = ["derive"] }
 tracing = "0.1.37"
-wayland-protocols-wlr = { version = "0.1.0", features = ["server"], optional = true}
-cosmic-config = { git = "https://github.com/pop-os/libcosmic" }
+wayland-protocols-wlr = { version = "0.2.0", features = ["server"], optional = true}
+cosmic-config = { git = "https://github.com/pop-os/libcosmic" , branch="theme-dark-light-switching" }
```

Comment:
This should be pointed at `master` now, right?

Reply:
Huh, was Github serving me an old version of the diff even though I had just refreshed the page? Whatever, everything here looks good.